src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -282,19 +277,15 @@ const ClientTemplateForm = ({ setErrorMessage, recordId, _labels, plantId, maxAc
           cellPhoneRepeatEncrypt: obj.clientMaster.cellPhone && obj.clientMaster.cellPhone,
           cellPhoneRepeat: obj.clientMaster.cellPhone,
           createdDate: obj.clientMaster.createdDate,
-
-          // expiryDate  :obj.clientMaster.expiryDate,
           flName: obj.clientMaster.flName,
           keyword: obj.clientMaster.keyword,
           otp: obj.clientMaster.otp,
-
-          // status: obj.clientMaster.status,
-          plantId: obj.clientMaster.plantId,
+          plantId: obj.plantId,
```

Comment:
no, this plantId field is always = 0 from the API. we don't need this one
we need the plantId from clientRemittance